### DISCUSSION TITLE
Remove Health Bars

### DISCUSSION THREAD

**Discussion by esser2002** [FakeFishGames/Barotrauma#5936] [Ideas and suggestions]
Health bars take away from immersion, and tension. Fighting an enemy monster, while knowing how much health it has left can become a simple numbers game, but not knowing whether an enemy is dead or a alive, high or low health makes every battle more tense, as it should be. The magic of hidden information should not be underestimated, and even the health system can even feel more complex and realistic, just by removing information, and letting imagination do some work. Changing visuals for damaged enemies would be nice, but the certainty provided by knowing the exact amount of health an enemy has removes from the overall combat experience, in my opinion.

**Comment by Rokvach** [comment]
Health bars has always been in the game, only now we've decided that bigger creatures health bars should be more visual as most of the time you fight them you won't be able to see the creatures more than couple seconds when they attack your submarine. So there was no way of telling how much health those types of creatures even had, I agree that it would of been more immersive to not see that information but I also feel like the encounter would of been a lot more boring without the health bars as the fight can drag for a very long time, especially if you are using some ammunition that isn't the best.

**Reply by esser2002** [reply]
I can see how it can be demotivating, and the lack of information can even be impeding to gameplay. 
Perhaps the best of both worlds could be achieved, by having an in-game tool to determine the health of attacking creatures. Perhaps a biotracker station on the ship could lock on nearby creatures, and would show the vitals of a creature it is locked onto. This would increase immersion, and promote teamwork, while still providing a means to determine the remaining health of attackers.

**Reply by PaladynMinak** [reply]
Removing ordinary ones from humans and lesser monsters, while keeping 'boss' health bars can be a compromise. Could also be as an option to toggle in campaign/server settings.

**Comment by James-E-A** [comment]
I'm not necessarily a fan of this idea, but it _could_ be nice if:

- it were nerfed to only be part of the Medical HUD
- the Medical HUD were buffed to work through periscopes

it already "feels powerful" to have a medical HUD giving you every last detail of your enemy's illnesses; making the gunner decide whether they want to be prepared for a hull breach, or have the edge and ability to give the captain status reports, would be a fun gameplay dynamic

plus, having it "restricted" like that would mean that they could then be free to make it **more visible**: over half the time, I don't even bother looking for it on anything larger than a Mudraptor because it's too puny and gets lost in combat. I just trust my learned ammo types and shoot until it's dead, which is OK but not as involved.

**Reply by GoodGuyJimmy** [reply]
That is a cool way of implementing this idea, either that or making the removal of health bars a server option. I personally like your idea the most though.

**Reply by MystLeissa** [reply]
I like the way of thinking that it should be a dedicated device that gives visual health bars, but I'm more in favor of adding it to a new item than trying to incorporate it into an existing one, the medical scanner HUD already feels OP, but that's mostly because there's no obvious choices besides it for a crew-member who doesn't wear much headgear (like the mechanic pseudo-captain, or the assistant :P)

**Comment by leonard84** [comment]
I am against a full removal, but it would be OK to have several immersion options for those that want less interface, so that they can turn it off in their settings without affect other players.

**Reply by GoodGuyJimmy** [reply]
I'd say it should be a server setting instead of a client thing, kinda like line of sight.

**Comment by someguythatplaysbarotrauma** [comment]
It would be better if there was just an option to turn them off

**Reply by someguythatplaysbarotrauma** [reply]
well i'm not on either side of this debate, I really just don't care

**Comment by cshingle** [comment]
This would be a nice server side option. The lack of health info would make the game harder. This sounds like exactly the type of thing experienced players could disable to for difficulty/role play reasons. Just leave them enabled by default for newer players.

**Comment by Hazmat616** [comment]
As long as the health scanner HUD still shows them while you can see a target yourself(not using periscope) I like this idea.
